.travis.yml

```diff
@@ -14,3 +14,11 @@ script:
     - sh build.sh
     # Invoke ci script too
     - sh contrib/ci.sh
+after_success:
+    - if [[ "$TRAVIS_TAG" != null ]]; then 
```

Comment:
During testing this did not work for me. This appears to work:

```
if [ ! -s "$TRAVIS_TAG" ] ; then
    echo "This build will be published under the tag: ${TRAVIS_TAG}"
fi
```

Reply:
Strange. I tested it with my Travis account and it worked fine, but I updated it to match what you show